ramls/interface.raml

```diff
@@ -51,3 +52,43 @@ resourceTypes:
         exampleItem: !include acq-models/mod-orgs/examples/interface_get.sample
         schema: interface
     is: [validate]
+
+    /credentials:
+      displayName: Purchace Order Lines
+      description: Manage purchase order (PO) lines
```

Comment:
please change to a valid description and display name

Reply:
fixed